PlutoHelperAgent/AppDelegate.m

```diff
@@ -24,9 +25,26 @@ - (id)init
  		  forEventClass:kInternetEventClass
      andEventID:kAEGetURL];
     
+    [self setup_defaults];
+    [SharedFunctions login_to_project_server];
+    
     return self;
 }
 
+
+- (void)setup_defaults
+
+{
+    
+    NSString *userDefaultsValuesPath = [@"~/Library/Preferences/com.GNM.PlutoHelperAgent.plist" stringByExpandingTildeInPath];
+    NSDictionary *userDefaultsValuesDict;
+    userDefaultsValuesDict=[NSDictionary dictionaryWithContentsOfFile:userDefaultsValuesPath];
+    [[NSUserDefaults standardUserDefaults] registerDefaults:userDefaultsValuesDict];
+    NSLog(@"Defaults %@", userDefaultsValuesDict);
```

Comment:
I don't think you actually need to explicitly load a plist like this - I think that `standardUserDefaults` should do that for you? @blishen could you confirm?

Reply:
As fas as I know you do need to do it like this. It would not know that file to load otherwise. The file name is just something I made up.

Reply:
it actually uses multiple files, starting from a system library location and moving into the user profile.  I think that the filename is based on the app name and domain that are found in `Info.plist`